kitchen.dokken.yml

```diff
@@ -63,16 +53,16 @@ platforms:
     image: dokken/fedora-latest
     pid_one_command: /usr/lib/systemd/systemd
 
-- name: ubuntu-16.04
+- name: ubuntu-18.04
```

Comment:
We still need 16.04 as well here

Reply:
I can add them back in, thought policy was latest and 1 previous OS and chef-infra client versions.

Reply:
Prematurely removing support for older clients makes it very hard for users to upgrade from legacy versions of chef-client so we try to keep support back as far as reasonably possible.